### PR TITLE
Bugfix/modify use tappay

### PR DESCRIPTION
**Pull request by HLTVProxy**
- modify useTappay setupSDK use useMemo

### CODE REVIEW

**Comment by HLTVProxy** [comment]
[(3) 【頁面管理】學米頁面管理蓋的頁面會噴很多error](https://trello.com/c/OJFTjcrN/3809-3-%E3%80%90%E9%A0%81%E9%9D%A2%E7%AE%A1%E7%90%86%E3%80%91%E5%AD%B8%E7%B1%B3%E9%A0%81%E9%9D%A2%E7%AE%A1%E7%90%86%E8%93%8B%E7%9A%84%E9%A0%81%E9%9D%A2%E6%9C%83%E5%99%B4%E5%BE%88%E5%A4%9Aerror)

File: src/hooks/util.ts [hunk 1]
```diff
@@ -8,14 +8,16 @@ export const useTappay = () => {
...
+  useMemo(() => {
+    settings['tappay.app_id'] &&
+      settings['tappay.app_key'] &&
+      TPDirect &&
+      TPDirect.setupSDK(
+        settings['tappay.app_id'],
+        settings['tappay.app_key'],
+        settings['tappay.dry_run'] === 'true' ? 'sandbox' : 'production',
+      )
+  }, [TPDirect, settings])
```
Comment (urfit-service) [inline src/hooks/util.ts:20]: useMemo doesn't seem to be made for you to use like this...
ref: https://zh-hant.reactjs.org/docs/hooks-reference.html#usememo

Comment (sunnylochun) [inline src/hooks/util.ts:20]: useMemo doesn't seem to be made for you to use like this
ref: https://reactjs.org/docs/hooks-reference.html#usememo

Comment (HLTVProxy) [reply src/hooks/util.ts]: had resolved in new commit. thx

**Review by sunnylochun — APPROVED** [review]

**Pull request merged**